src/api/rateArticleApi.js

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+import instance from "./axiosConfig";
+
+const token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VybmFtZSI6ImRlbm8iLCJyZWZyZXNoX3Rva2VuIjpmYWxzZSwiaWF0IjoxNTQ4MzIyNzAyLCJuYmYiOjE1NDgzMjI0MDIsImV4cCI6MTU0ODMyNjMwMn0.-skCSxs27w1iPZw2CtH9TF-y02Ybz65QYhhe31hn0Kk";
+axios.defaults.headers.common.Authorization = token;
+
+export default class rateArticleApi {
+  static rateArticle(rated, slug) {
+    return instance
+      .post(
+        `/api/article/${slug}/rate`,
+        {
+          article: {
+            rate: rated
+          }
+        },
+        {
+          headers: {
+            Authorization: `Token ${token}`
```

Comment:
why are we using a hard coded token?

Reply:
This was before integration but I have fixed this.